tests/friendship_server.py

```diff
@@ -0,0 +1,98 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from concurrent import futures
+from typing import List
+
+import grpc
+
+from friendship_pb2 import Person, CreateResult, PersonEmail, CommonFriendsResult, FriendshipResponse
+from friendship_pb2_grpc import FriendshipServicer, add_FriendshipServicer_to_server
+
+
+class FriendShipService(FriendshipServicer):
+    """
+    Implementation of the gRPC FriendshipServicer.
+    See the friendship.proto definition for more info.
+    """
+    def __init__(self):
+        self.people = list()
+
+    def Create(self, request, context):
+        person: Person = request
```

Comment:
We still have to be Python 2.7 compatible

```suggestion
        person = request  # type: Person
```